mne/io/brainvision/brainvision.py

```diff
@@ -296,9 +298,18 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
                       "not the '%s' file." % ext)
     with open(vhdr_fname, 'rb') as f:
         # extract the first section to resemble a cfg
-        header = f.readline().decode('utf-8').strip()
+        header = f.readline()
+        try:
+            header = header.decode('utf-8').strip()
+        except UnicodeDecodeError:
+            header = header.decode('ISO-8859-1')
```

Comment:
So to be complete I guess we'd want to 1) parse header, and 2) fall back to UTF-8 then 3) fall back to latin-1 (and let's use that name since it's more human-readable)?